package.json

```diff
@@ -47,38 +55,46 @@
   "homepage": "https://github.com/ERC725Alliance/erc725.js#readme",
   "devDependencies": {
     "@biomejs/biome": "^1.6.4",
-    "@types/chai": "^4.3.4",
-    "@types/isomorphic-fetch": "^0.0.36",
-    "@types/jest": "^27.5.2",
-    "@types/mocha": "^10.0.1",
-    "@types/node": "^18.11.19",
-    "@types/sinon": "^10.0.13",
-    "@typescript-eslint/eslint-plugin": "^5.50.0",
-    "@typescript-eslint/parser": "^5.50.0",
-    "assert": "^2.0.0",
-    "chai": "^4.3.7",
-    "eslint": "^8.33.0",
+    "@types/chai": "~4.3.11",
+    "@types/jest": "~27.5.2",
+    "@types/mocha": "~10.0.6",
+    "@types/node": "^20.11.16",
+    "@types/sinon": "~10.0.20",
+    "@typescript-eslint/eslint-plugin": "^6.21.0",
+    "@typescript-eslint/parser": "^6.21.0",
+    "assert": "^2.1.0",
+    "chai": "~4.4.1",
+    "cross-env": "^7.0.3",
+    "eslint": "^8.56.0",
     "eslint-config-airbnb-base": "^15.0.0",
-    "eslint-config-prettier": "^8.6.0",
-    "eslint-plugin-import": "^2.27.5",
-    "eslint-plugin-prettier": "^5.0.0",
-    "isomorphic-fetch": "^3.0.0",
-    "jest": "^27.5.1",
-    "mocha": "^10.0.0",
+    "eslint-config-prettier": "^9.1.0",
+    "eslint-plugin-import": "^2.29.1",
+    "eslint-plugin-prettier": "^5.1.3",
+    "esm": "^3.2.25",
+    "jest": "^29.7.0",
+    "mocha": "^10.2.0",
     "npm-run-all": "^4.1.5",
     "nyc": "^15.1.0",
-    "prettier": "^3.0.0",
-    "sinon": "^15.0.1",
+    "prettier": "^3.2.5",
+    "sinon": "^17.0.1",
     "standard-version": "^9.5.0",
-    "ts-node": "^10.9.1",
-    "typescript": "^4.9.5",
-    "web3": "^1.10.0"
+    "ts-node": "^10.9.2",
+    "typescript": "^5.3.3"
   },
   "dependencies": {
-    "add": "^2.0.6",
-    "ethereumjs-util": "^7.1.5",
-    "web3-eth-abi": "^1.10.0",
-    "web3-providers-http": "^1.10.0",
-    "web3-utils": "^1.10.0"
+    "cross-fetch": "^4.0.0",
+    "global-agent": "^3.0.0",
+    "msw": "^2.3.1",
```

Comment:
why do we have these top 3 now in `"dependencies"`? Is it necessary for the lib to run?
Asking as the library is small, we we want to keep to a minimum and wondering if it will increase the size.

Reply:
Hmm I forget "global-agent", but "cross-fetch" is because of the data-providers so we don't include fetch two times. Checking on global-agent.

Reply:
Removing global-agent. This is a thing for debugging similar to fiddler on windows. It allows you to enable proxies for apps so you can watch the traffic.